src/Model/ConsoleModel.php

```diff
@@ -31,10 +31,12 @@ class ConsoleModel extends ViewModel
      * Set error level to return after the application ends.
      *
      * @param int $errorLevel
+     * @return $this
```

Comment:
Please use `self` here.

Reply:
@froschdesign why `self`? I think `$this` is the most right here, as we return the same instance, not the new one. I think this is also what is suggested in PSR-5 draft: https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md - check the bottom of the document: points 13-15.

Reply:
@webimpress 
I would like to remove it completely! (`>=7.1`)

Reply:
@froschdesign Removing them is a different story and this is not answer for my question, why you think `self` is more appropriate here ;-) 
As I understand fluent interfaces makes sens in some cases, and not always they are "evil".
https://ocramius.github.io/blog/fluent-interfaces-are-evil/

Reply:
@webimpress 
No, I mean remove the return type in the DocBlock.

Reply:
@froschdesign I think what was decided in ZF is to remove them (PHPDocs) only if they are not providing any additional information. And in that case, even if we add RTH `: self` still `@return $this` gives us more information about returned object (as described - in quoted before - PSR-5 draft).

Reply:
Then I will stop the discussion here. It makes no sense without any coding standard.

Reply:
ping https://github.com/zendframework/zend-coding-standard/pull/5